contracts/0.8.9/DepositSecurityModule.sol

```diff
@@ -316,7 +323,7 @@ contract DepositSecurityModule {
         int256 guardianIndex = _getGuardianIndex(msg.sender);
 
         if (guardianIndex == -1) {
-            bytes32 msgHash = keccak256(abi.encodePacked(PAUSE_MESSAGE_PREFIX, blockNumber, stakingModuleId));
+            bytes32 msgHash = keccak256(abi.encodePacked(PAUSE_MESSAGE_PREFIX, blockNumber, uint24(stakingModuleId)));
```

Comment:
it's a bit inconsistent maybe to have such a type cast here because `blockNumber` is `uint256`. for example
maybe use `uint256` for `stakingModuleId` too?

Reply:
done